pom.xml

```diff
@@ -145,5 +169,61 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>release-sign-artifacts</id>
+            <!-- for release -->
+            <!-- mvn -DperformRelease=true deploy -->
+            <activation>
+                <property>
+                    <name>performRelease</name>
+                    <value>true</value>
+                </property>
+            </activation>
```

Comment:
- [ ] It is difficult to use because there are a lot of the selection way of maven profiles. Let's standardize these.